src/libs/react-utils.ts

```diff
@@ -161,10 +176,18 @@ export const useStore = theStore => {
  * @param [id]: Optional: The ID of the component if allowId is true
  * @param [tooltip] Optional: The tooltip provided to the component if allowTooltip is true
  */
-export const useLabelAssert = (componentName, {
-  allowLabelledBy = true, allowId = false, allowTooltip = false, allowContent = false,
-  'aria-label': ariaLabel, 'aria-labelledby': ariaLabelledBy, id, tooltip
-}) => {
+export const useLabelAssert = (componentName: string, options: UseLabelAssertOptions) => {
```

Comment:
Moving destructuring to a separate line makes the function/type signature easy to see.